<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Penrose Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Schéma aigu :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="acute_pattern">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Schéma obtu :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="obtuse_pattern">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Double découpe :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="double_cut">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Itération :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="level">
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Dessiner
les lignes :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="draw_lines">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="action_propos"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_propos">
   <property name="text">
    <string>À propos...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>level</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>draw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acute_pattern</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>regenerate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>obtuse_pattern</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>regenerate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>double_cut</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>regenerate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>draw_lines</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>draw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_propos</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>a_propos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>draw()</slot>
  <slot>regenerate()</slot>
  <slot>a_propos()</slot>
 </slots>
</ui>
